I added a few pages translated into Turkish. (#1038)
withastro/starlight: docs/src/content/docs/tr/guides/i18n.mdx

---
title: Uluslararasılaştırma (i18n)
description: Starlight sitesinin çoklu dil desteğini etkinleştirmek için nasıl yapılandırılacağını öğrenin.
---

> import FileTree from '../../../components/file-tree.astro';

Reviewer: ```suggestion
import FileTree from '../../../../components/file-tree.astro';
```

Desteklediğiniz ek diller için çevirileri sağlayabilir veya varsayılan etiketlerimizi `i18n` veri koleksiyonu aracılığıyla geçersiz kılabilirsiniz.

1. Eğer zaten yapılandırılmamışsa `src/content/config.ts` içinde `i18n` veri koleksiyonunu yapılandırın:

   ```js
   // src/content/config.ts
   import { defineCollection } from 'astro:content';
   import { docsSchema, i18nSchema } from '@astrojs/starlight/schema';

   export const collections = {
   	docs: defineCollection({ schema: docsSchema() }),
   	i18n: defineCollection({ type: 'data', schema: i18nSchema() }),
   };
   ```

2. Her ek dil için `src/content/i18n/` içinde bir JSON dosyası oluşturun. Örneğin, bu Arapça ve Basitleştirilmiş Çince için çeviri dosyalarını ekler:

   <FileTree>

   - src/
     - content/
       - i18n/
         - ar.json
         - zh-CN.json

   </FileTree>

3. Çevirmek istediğiniz anahtarlar için JSON dosyalarına çeviriler ekleyin. Sadece değerleri çevirin ve anahtarları İngilizce olarak bırakın (örneğin, "search.label": "Arama").

   İşte Starlight'ın varsayılan İngilizce değerleri:

   ```json

>    {
>    	"skipLink.label": "İçeriğe Git",
>    	"search.label": "Ara",
>    	"search.shortcutLabel": "(Aramak için / tuşuna basın)",
>    	"search.cancelLabel": "İptal",
>    	"search.devWarning": "Arama yalnızca üretim sürümlerinde kullanılabilir. \nYerel olarak test etmek için siteyi oluşturup önizlemeyi deneyin.",
>    	"themeSelect.accessibleLabel": "Tema seçin",
>    	"themeSelect.dark": "Koyu",
>    	"themeSelect.light": "Açık",
>    	"themeSelect.auto": "Açık",
>    	"languageSelect.accessibleLabel": "Dil seçin",
>    	"menuButton.accessibleLabel": "Menü",
>    	"sidebarNav.accessibleLabel": "Ana",
>    	"tableOfContents.onThisPage": "Bu sayfada",
>    	"tableOfContents.overview": "Genel Bakış",
>    	"i18n.untranslatedContent": "Bu içerik dilinizde henüz mevcut değil.",
>    	"page.editLink": "Sayfayı düzenle",
>    	"page.lastUpdated": "Son Güncelleme:",
>    	"page.previousLink": "Sonraki",
>    	"page.nextLink": "Önceki",
>    	"404.text": "Sayfa bulunamadı. URL yi kontrol edin veya arama çubuğunu deneyin."
>    }

Reviewer 2: This should be left in English as indicated by the line immediately above this code block:

> İşte Starlight'ın varsayılan İngilizce değerleri:

```suggestion
   {
   	"skipLink.label": "Skip to content",
   	"search.label": "Search",
   	"search.shortcutLabel": "(Press / to Search)",
   	"search.cancelLabel": "Cancel",
   	"search.devWarning": "Search is only available in production builds. \nTry building and previewing the site to test it out locally.",
   	"themeSelect.accessibleLabel": "Select theme",
   	"themeSelect.dark": "Dark",
   	"themeSelect.light": "Light",
   	"themeSelect.auto": "Auto",
   	"languageSelect.accessibleLabel": "Select language",
   	"menuButton.accessibleLabel": "Menu",
   	"sidebarNav.accessibleLabel": "Main",
   	"tableOfContents.onThisPage": "On this page",
   	"tableOfContents.overview": "Overview",
   	"i18n.untranslatedContent": "This content is not available in your language yet.",
   	"page.editLink": "Edit page",
   	"page.lastUpdated": "Last updated:",
   	"page.previousLink": "Next",
   	"page.nextLink": "Previous",
   	"404.text": "Page not found. Check the URL or try using the search bar."
   }
```

   ```

	Starlight'ın arama penceresi [Pagefind](https://pagefind.app/) kütüphanesi tarafından desteklenmektedir.
	Aynı JSON dosyasını kullanarak Pagefind'in kullanıcı arayüzü için çevirileri ayarlayabilirsiniz. `pagefind` anahtarları kullanılarak:

>    ```json
> 	{
> 	"pagefind.clear_search": "Temizle",
> 	"pagefind.load_more": "Daha fazla sonuç yükle",
> 	"pagefind.search_label": "Bu siteyi ara",
> 	"pagefind.filters_label": "Filtreler",
> 	"pagefind.zero_results": "[ARAMA_TERIMI] için sonuç bulunamadı",
> 	"pagefind.many_results": "[ARAMA_TERIMI] için [SAYI] sonuç",
> 	"pagefind.one_result": "[ARAMA_TERIMI] için [SAYI] sonuç",
> 	"pagefind.alt_search": "[ARAMA_TERIMI] için sonuç bulunamadı. Bunun yerine [FARKLI_TERIM] sonuçları gösteriliyor",
> 	"pagefind.search_suggestion": "[ARAMA_TERIMI] için sonuç bulunamadı. Aşağıdaki aramalardan birini deneyin:",
> 	"pagefind.searching": "[ARAMA_TERIMI] aranıyor..."
> 	}
> 
>    ```

Reviewer 2: Same here:

````suggestion
   ```json
   {
   	"pagefind.clear_search": "Clear",
   	"pagefind.load_more": "Load more results",
   	"pagefind.search_label": "Search this site",
   	"pagefind.filters_label": "Filters",
   	"pagefind.zero_results": "No results for [SEARCH_TERM]",
   	"pagefind.many_results": "[COUNT] results for [SEARCH_TERM]",
   	"pagefind.one_result": "[COUNT] result for [SEARCH_TERM]",
   	"pagefind.alt_search": "No results for [SEARCH_TERM]. Showing results for [DIFFERENT_TERM] instead",
   	"pagefind.search_suggestion": "No results for [SEARCH_TERM]. Try one of the following searches:",
   	"pagefind.searching": "Searching for [SEARCH_TERM]..."
   }
   ```
````
